index.js

```diff
@@ -4,9 +4,24 @@ let instances = {};
 
 function DB() {
   this.pool = null;
+  this.sessionConfig = null;
   this.debug = false;
 }
 
+function setSessionConfig(connection, config) {
+  if(!config) {
+    return;
+  }
+
+  if(this.debug) {
+    console.log('Set session config');
```

Comment:
I think it would be nice to also log the actual session config here.

Reply:
done